develop-docs/self-hosted/experimental/monitoring.mdx

```diff
@@ -0,0 +1,182 @@
+    DOGSTATSD_PORT: 8125
+
+# The rest of your Docker Compose file
+```
+
+### Relay
+
+You can configure Relay to send metrics to Statsd server by configuring your `relay/config.yml` file:
+
+```yaml
+# The rest of your config.yml file
+metrics:
+  statsd: "statsd.yourcompany.com:8125"
+  # prefix: "sentry.relay" # Adjust this to your needs, default is "sentry.relay"
+  # sample_rate: 1.0 # Adjust this to your needs, default is 1.0
+  # `periodic_secs` is the interval for periodic metrics emitted from Relay.
+  # Setting it to `0` seconds disables the periodic metrics.
+  # periodic_secs: 5
+```
+
+### Symbolicator
+
+You can configure Symbolicator to send metrics to Statsd server by configuring your `symbolicator/config.yml` file:
+
+```yaml
+# The rest of your config.yml file
+metrics:
+  statsd: "statsd.yourcompany.com:8125"
+  prefix: "sentry.symbolicator" # Adjust this to your needs, default is "symbolicator"
+```
+
+## Sentry dependencies
+
+Sentry and its' surrounding services have some dependencies like Redis, ClickHouse, Kafka, etc, that you can monitor with your monitoring system. The easiest way to do so is through Prometheus exporters. Although there are many exporters available, for reference purposes, you can use this setup below.
+
+Modify your `docker-compose.yml` file to add the following:
+
+```yaml
+services:
+  # The rest of your Docker Compose file
+  prometheus-postgresql-exporter:
+    <<: *restart_policy
+    image: quay.io/prometheuscommunity/postgres-exporter
+    ports:
+      - "127.0.0.1:9187:9187"
+    environment:
+      DATA_SOURCE_URI: "postgres:5432/postgres?sslmode=disable"
+      DATA_SOURCE_USER: "postgres"
+  prometheus-redis-exporter:
+    <<: *restart_policy
+    image: oliver006/redis_exporter
+    ports:
+      - "127.0.0.1:9121:9121"
+    environment:
+      REDIS_ADDR: "redis:6379"
+  prometheus-kafka-exporter:
+    <<: *restart_policy
+    image: danielqsj/kafka-exporter
+    ports:
+      - "127.0.0.1:9308:9308"
+    command: ["--kafka.server=kafka:9092"]
+  prometheus-memcached-exporter:
+    <<: *restart_policy
+    image: quay.io/prometheus/memcached-exporter
+    ports:
+      - "127.0.0.1:9150:9150"
+    command: ["--memcached.address=memcached:11211"]
+```
+
+Modify your `clickhouse/config.xml` file to add the following:
+
+```xml
+<clickhouse>
+    <!-- The rest of your ClickHouse configuration -->
+    <prometheus>
+        <endpoint>/metrics</endpoint>
+        <port>9363</port>
+        <metrics>true</metrics>
+        <events>true</events>
+        <asynchronous_metrics>true</asynchronous_metrics>
+        <errors>true</errors>
+    </prometheus>
+</clickhouse>
+```
+
+Up to this point, all metrics are exposed only on `localhost`. If you don't want your Sentry installation to expose any other ports, you can remove every `ports` directive on the `docker-compose.yml` file and route everything through the `nginx` container, by modifying your `nginx.conf` file to the following:
```

Comment:
There is also one reason why I don't provide monitoring the Nginx container, because I don't think that the `stub_status` module is not included on the Nginx's Alpine Docker image: https://nginx.org/en/docs/http/ngx_http_stub_status_module.html